dev-doc/RegenerateLanguages.md

```diff
@@ -0,0 +1,45 @@
+# Regenerate Languages
+
+This document describes things that need to be done when regenerate the test- and production languages.
+
+1. **xtext-core:** execute the `generateTestLanguages` Gradle task
+```
+./gradlew -PuseJenkinsSnapshots=true generateTestLanguages
+```
+
+1. **xtext-lib:** nothing to do
```

Comment:
Can be removed, if there is nothing to do.

Reply:
Fixed.